docs/content/latest/api/ycql/batch.md

```diff
@@ -0,0 +1,173 @@
+---
+title: BATCH requests [YCQL]
+headerTitle: BATCH
+linkTitle: BATCH
+summary: Execute multiple DML in 1 request
+description: Use batch to update multiple rows in 1 request.
+menu:
+  latest:
+    parent: api-cassandra
+    weight: 19991
+aliases:
+  - /latest/api/ycql/batch2
```

Comment:
why `batch2`?

Reply:
It's no longer needed because docs have been updated.